docs/src/main/sphinx/release/release-444.md

```diff
@@ -0,0 +1,79 @@
+# Release 444 (29 Mar 2024)
+
+## General
+
+* Prevent planning failures due to invalid expression equivalence from
```

Comment:
@martint just a suggestions.. probably should be improved.

Reply:
“Fix failure for queries containing large numbers of LIKE terms in boolean expressions”

Reply:
Updated